src/components/Article.jsx

```diff
@@ -1 +1,28 @@
-// export const Article = () => ();
+import React from 'react';
+
+export function Article({ title, date, text }) {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="box">
```

Comment:
you can use specific html tags
```suggestion
    <article className="box">
```